### PR TITLE
Remove legacy API from common test utils

### PR DESCRIPTION
**Pull request by ilyachur**
### Details:
 - *item1*
 - *...*

### Tickets:
 - *ticket-id*

### CODE REVIEW

File: src/frontends/onnx/tests/onnx_import.in.cpp [hunk 1]
```diff
@@ -25,6 +25,7 @@
 // clang-format on
 #include "common_test_utils/file_utils.hpp"
 #include "common_test_utils/ov_test_utils.hpp"
+#include "ngraph/file_util.hpp"
```
[the remark this answers is not in the document]
Comment (ilyachur) [reply src/frontends/onnx/tests/onnx_import.in.cpp:28]: We have a dedicated ticket foe ONNX tests

File: src/tests/test_utils/common_test_utils/src/data_utils.cpp [hunk 2]
```diff
@@ -4,10 +4,9 @@
 
 #include "common_test_utils/data_utils.hpp"
 
+#include "blob_factory.hpp"
```
[the remark this answers is not in the document]
Comment (ilyachur) [reply src/tests/test_utils/common_test_utils/src/data_utils.cpp:7]: From my point of view it should be removed with legacy API, and we will avoid these methods after tests migration to 2.0 API

File: src/frontends/onnx/tests/onnx_import_com_microsoft.in.cpp [hunk 3]
```diff
@@ -18,6 +18,7 @@
 #include "common_test_utils/test_case.hpp"
 #include "onnx_import/onnx.hpp"
 #include "onnx_utils.hpp"
+#include "ngraph/file_util.hpp"
```
Comment (praasz) [inline src/frontends/onnx/tests/onnx_import_com_microsoft.in.cpp:21]: As removing old API it should be added?

Comment (ilyachur) [reply src/frontends/onnx/tests/onnx_import_com_microsoft.in.cpp:21]: We have the same API in openvino::util library.

ONNX tests should be migrated to new API by ONNX team

**Review by praasz — APPROVED** [review]

**Pull request merged**